packages/server/src/parser.ts

```diff
@@ -112,13 +112,18 @@ function updateFrameButton(frameInfo: DeepPartial<OpenFrameResult>, key: string,
 	frameInfo.buttons = frameInfo.buttons || {};
 	const button = frameInfo.buttons[buttonIndex] || {};
 	if (maybeField) {
-		const field = maybeField as keyof OpenFrameButton;
+		const field = maybeField as keyof OpenFrameButton | 'post_url';
```

Comment:
Isn't `post_url` actually available on all buttons?

I think this is a bigger bug 😬 , and we should be returning it even if the action isn't `tx`

Reply:
Hmmm I don't think so; or at least it's specifically added to the `OpenFrameButton` type only in `tx` frames. are you suggesting that part might be wrong and we should update those types as well? Happy to do that as part of this PR if so.

Reply:
I believe it should also be available on `post` or `post_redirect` buttons as well

Reply:
Also, for the other OF field types we've been camelCasing the names in our returned values